### ISSUE TITLE
Native Unicode support

### ISSUE THREAD

**Issue by GoogleCodeExporter** [pennmush/pennmush#366]
```
I know this is a very big project. I would REALLY like to see pennmush natively 
support Unicode. This is important to anyone who has an interest in ASCII art.
```

Original issue reported on code.google.com by `adk...@drigon.com` on 21 Sep 2010 at 7:42
- Blocked on: #622
- Merged into: #622
